modules/dreamview/backend/sim_control/sim_control.cc

```diff
@@ -109,7 +109,7 @@ void SimControl::InitTimerAndIO() {
 
 void SimControl::Init(bool set_start_point, double start_velocity,
```

Comment:
The first argument set_start_point is no longer needed, remove it to make the function cleaner,  then test to make sure there is no problem with sim_control